frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/editReleaseInformation.jspf

```diff
@@ -38,21 +38,28 @@
         </td>
     </tr>
     <tr>
-        <td width="33%">
+        <td width="25%">
             <label class="textlabel stackedLabel" for="programminglanguages">Programming Languages</label>
             <input class="toplabelledInput" name="<portlet:namespace/><%=Component._Fields.LANGUAGES%>"
                    id="programminglanguages" type="text"
                    placeholder="e.g., Java,C++, C#,..."
                    value="<sw360:DisplayCollection value='${release.languages}' />"/>
         </td>
-        <td width="33%">
+        <td width="25%">
             <label class="textlabel stackedLabel" for="op_systems">Operating Systems</label>
             <input class="toplabelledInput" id="op_systems"
                    name="<portlet:namespace/><%=Component._Fields.OPERATING_SYSTEMS%>" type="text" align="middle"
                    placeholder="e.g.,Linux,MAC,Windows,..."
                    value="<sw360:DisplayCollection value="${release.operatingSystems}" />"/>
         </td>
-        <td width="33%">
+        <td width="25%">
```

Comment:
I don't wanna be rude, but did you look at your new GUI just once?
<img width="1514" alt="image" src="https://user-images.githubusercontent.com/27430165/58715532-605dc780-83c7-11e9-909a-5621b9063263.png">

Reply:
Yes i have. How would you like this to appear? 3/4/3/3 columns with the same width on the page?

Reply:
Why not just adding a row instead of a column? We have these three column layouts a lot and I don't see a reason why we should make an exception here. On the other masks all rows are filled except for the last one. So you would have to move the following fields accordingly.

Reply:
Clear. One question though. Where should the new text field "Software Platforms" be? At the end as the first element of the new row? After "operating systems" and then shift the rest by one?

Reply:
I think it would have been just fine, if you moved the additional field in order, resulting other subsequent fields to "line break"-"shift to next line" accordingly while maintaining a 3-column arrangement for the fields. dead space should be avoided.

In case you prefer a dedicated proposal: Why not moving the licenses field in a separate row and the CPE Id in front of release date so release date and download URL are next to each other.

Reply:
Very good. Thanks for the input.